src/constants/permissions.js

```diff
@@ -1,3 +1,12 @@
+// A description of permissions we're using aroung the project:
+// * canCreateArticle - the one with this permission is treated as an Author:
+//    he/she can create new articles and edit those for which he/she is an author
+//    (article's authorId matches).
+// * canManageArticles - the one with this permission is a contentManager:
+//    he/she can edit and remove existing articles bun CAN NOT create new.
+// * canManageUsers - the one with this permission can create, edit and remove
+//    and view the full list of users existing.
```

Comment:
TODO: use some standart for building docs from comment in the future, for example [`jsdoc`](https://github.com/jsdoc3/jsdoc)

Reply:
https://trello.com/c/dtNGlzjS/221-add-jsdoc-or-other-tool-for-docs-generation